packages/encoding/src/decoding/decodeDexOutput.ts

```diff
@@ -0,0 +1,50 @@
+import { PedersenHash } from '@explorer/types'
+
+import { StarkExDexOutput } from '../OnChainData'
+import { ByteReader } from './ByteReader'
+
+export function decodeDexOutput(data: string): StarkExDexOutput {
+  const reader = new ByteReader(data)
+
+  const configCode = reader.readBigInt(32)
+  const initialValidiumVaultRoot = PedersenHash(reader.readHex(32))
+  const finalValidiumVaultRoot = PedersenHash(reader.readHex(32))
+  const initialRollupVaultRoot = PedersenHash(reader.readHex(32))
+  const finalRollupVaultRoot = PedersenHash(reader.readHex(32))
+  const initialOrderRoot = PedersenHash(reader.readHex(32))
+  const finalOrderRoot = PedersenHash(reader.readHex(32))
+  const globalExpirationTimestamp = reader.readBigInt(32)
```

Comment:
Probably it should be `readNumber` unless we're using `bigint` for timestamps everywhere else.

Reply:
`minimumExpirationTimestamp` in `decodeFirstPage` is also read as `bigint` but `timestamp` in `decodeState` is read as `number`

Reply:
I see. In that case I'd use number here and in the future also modify minimumExpirationTimestamp to be a number too

Reply:
DOne